### ISSUE TITLE
How to install StoryTeller?

### ISSUE THREAD

**Issue by dawei03896** [jaketae/storyteller#3]
Good work！ But how to install StoryTeller? the project may be missing something?

**Comment by jaketae** [comment]
Hello @dawei03896, thanks for opening this issue. 

Currently, StoryTeller is not a package on PyPI and needs to be installed by cloning the repository, as outlined in the [README](https://github.com/jaketae/storyteller/blob/master/README.md).

```
$ git clone https://github.com/jaketae/storyteller.git
$ pip install --upgrade pip wheel # optionally update pip
$ pip install .
```

I will try to find time to make it into a `pip`-installable library in the near future. Let me know if you have further questions!

**Comment by dawei03896** [comment]
Thank you very much for your reply, I have installed it, it very nice.

**Comment by jaketae** [comment]
Glad to hear that it worked! I'll close this for now, but feel free to comment or re-open this issue if you run into problems.

**Comment by cvDreamer** [comment]
I followed the instruction to run "pip install storyteller-core" and found the following error:

error: subprocess-exited-with-error . python setup.py egg_info did not run successfully. exit code: 1,

when i try to intall myself by running 'pip install .', i got the following error:

'ModuleNotFoundError: No module named 'poetry'

I tried 'pip install poetry' but did not help.

My python version is 3.10.9 which should be satisfying the requirements.
